astropy/modeling/powerlaws.py

```diff
@@ -390,7 +390,7 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
 
 class ExponentialCutoffPowerLaw1D(Fittable1DModel):
     """
-    One dimensional power law model with an exponential cutoff.
+    One dimensional power law model with an exponential cutoff (also referred to as the Schecter function).
```

Comment:
Nitpick: Your addition makes the line too long. Would be nicer to have it on the next line.